django_jinja/backend.py

```diff
@@ -288,15 +287,15 @@ def get_template(self, template_name):
             else:
                 exc = TemplateDoesNotExist(exc.name, backend=self)
 
-            six.reraise(
+            utils.reraise(
```

Comment:
If dropping python2, could this reraise now be written as `raise exc from oldexc`?

(same below)

Reply:
Probably yes, however I believe this one alters the traceback `sys.exc_info()[2]`